test/e2e/grpc_test.go

```diff
@@ -214,14 +213,13 @@ func TestGRPCStreamingPing(t *testing.T) {
 }
 
 func waitForActivatorEPS(resources *v1a1test.ResourceObjects, clients *test.Clients) error {
-	aeps, err := clients.KubeClient.Kube.CoreV1().Endpoints(
-		system.Namespace()).Get(networking.ActivatorServiceName, metav1.GetOptions{})
-	if err != nil {
-		return fmt.Errorf("error getting activator endpoints: %v", err)
-	}
-
 	// Wait for the endpoints to equalize.
 	return wait.Poll(250*time.Millisecond, time.Minute, func() (bool, error) {
+		aeps, err := clients.KubeClient.Kube.CoreV1().Endpoints(
```

Comment:
does is make sense to cache `aeps` to avoid repeated query?

Reply:
No, this is the source of the flake since that changes.

Reply:
(aeps changes due to autoscaling of activator)

Reply:
I see, should there be a comment instead?

Reply:
Done